dashboard/app/views/projects/index.html.haml

```diff
@@ -0,0 +1,74 @@
+-if rack_env == :development
+  %script{src: "/shared/js/angular/angular.js"}
+  %script{src: "/shared/js/angular-route/angular-route.js"}
+  %script{src: "/shared/js/angular-resource/angular-resource.js"}
+-else
+  %script{src: "/shared/js/angular/angular.min.js"}
+  %script{src: "/shared/js/angular-route/angular-route.min.js"}
+  %script{src: "/shared/js/angular-resource/angular-resource.min.js"}
+
+%div{'ng-app' => 'projectsApp', 'ng-view' => ''}
+
+:javascript
+  'use strict';
+
+  // Declare app level module which depends on filters, and services
+  angular.module('projectsApp', [
+    'ngRoute',
+    'ngResource',
+    'projectsApp.controllers',
+    'projectsApp.services'
+  ]).config(['$routeProvider', function($routeProvider) {
+    $routeProvider.when('/',
+                        {templateUrl: '/projects/projects',
+                         controller: 'ProjectsController'});
+    $routeProvider.otherwise({redirectTo: '/'});
+  }]);
+
+  // SERVICES
+  var services = angular.module('projectsApp.services', []).
+    value('version', '0.1');
```

Comment:
Nit: prefer line continuation indent @ 4 spaces (same comment for line 59 & 62).